### PR TITLE
card_thermostat: Handle high/low temperature, external climate fan, and color for cooling

### PR DESCRIPTION
**Pull request by finder39**
This diff adds three main new features for the `card_thermostat` card:
1. Add the handling of `target_temp_high` & `target_temp_low` for "heat_cool" settings that allow a high/low temperature to be set.
2. Add the ability for fan entity to be given as a variable (used when Nest is added via Homebridge Integration).
3. Color the card icon blue when AC is running (while keeping red for heating) instead of always being red when either is running.

Showing high/low temperature and blue icon when AC is running:
card_themostat_with_high_low

### CODE REVIEW

**Comment by finder39** [comment]
The failed Code Quality Check seems to be from unrelated files

**Comment by finder39** [comment]
@CM000n would love a review if you have a chance :) Should be a great enhancement for people

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_thermostat.yaml [hunk 1]
```diff
@@ -567,14 +744,44 @@ card_thermostat:
...
+                      - background-color: "rgba(var(--color-theme),0.15)"
           fan_only:
             card:
               type: "custom:button-card"
               template: "widget_icon"
               icon: "mdi:fan"
               tap_action:
                 action: "call-service"
-                service: "climate.set_hvac_mode"
+                service: "climate.set_hvac_mode]"
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_thermostat.yaml:784]: Trailing bracket?

**Comment by basbruss** [comment]
Hi @finder39,

I have made some remarks to the code to be consist with the theme styling. Could you also redirect the PR to the `release` branch as this are adaptations to an official card: https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md

**Comment by finder39** [comment]
Closing this in favor of #596, which is built off the correct branch

**Pull request closed without merging**